File: c++/gles2/java/GisPolygon.java
package zydz.gis.gles2;

import android.opengl.GLES20;

class GisPolygon  extends GisLines{  
	final String TAG = "GisPolygon.";
	
    public GisPolygon() {
    	super();
    }

    @Override
    public void draw(float[][] points, float[] rgb){        
    	super.draw(points, rgb, GLES20.GL_LINE_LOOP);
    }
}

File: c++/gles2/src/graph/GisPolygon.cpp
#include "GisPolygon.h"

GisPolygon::GisPolygon(const std::vector<GisPoint> &points, GisColor c)
    :GisLines(points, c)
{
} 

GisPolygon::GisPolygon(void)
    :GisLines()
{
}

GisPolygon::~GisPolygon(void)
{
    ;
}

void GisPolygon::Draw(void)
{
    GisPrimitives::Draw(GL_LINE_LOOP);
}

